pkg/crc/machine/virtualmachine.go

```diff
@@ -89,7 +89,7 @@ func (vm *virtualMachine) State() (state.State, error) {
 
 func (vm *virtualMachine) IP() (string, error) {
 	if vm.vsock {
-		return "127.0.0.1", nil
+		return VsockPrivateAddress(), nil
```

Comment:
similar to https://github.com/crc-org/crc/pull/3337/files#r1302624090 this should also report the WSL vswitch ip address only when `wsl-host-access` is enabled, but currently its not considering the config option

Reply:
It'd be easier to toggle this with an env variable instead of the config option, so i suggest instead of having a config option `wsl-host-access` we can depend on a env variable `CRC_WSL_HOST_ACCESS` which should toggle this behavior, wdyt @gbraad ?

Reply:
I don't know from where to where the connections would be allowed, but crc has a `host-network-access                        Allow TCP/IP connections from the CRC VM to services running on the host` config option, let's try to get the naming of the new option consistent/not confusing with this preexisting option.

Reply:
The comm. is to be allowed from WSL to crc VM, so that the `oc` commands work from the WSL terminal

i mistakenly made the config key `wsl-host-access` but previously `wsl2-network-access` was suggested in https://github.com/crc-org/crc/pull/3337#issuecomment-1248904591

on the other hand `host-network-access` allows for communication from inside the CRC VM to the host